pyuvsim/simsetup.py

```diff
@@ -44,12 +45,84 @@ def _parse_layout_csv(layout_csv):
                          dtype=dt.dtype)
 
 
-def read_votable_catalog(gleam_votable):
+def _array_to_sourcelist(catalog_table, lst_array=None, time_array=None, latitude_deg=None, horizon_buffer=0.04364, min_flux=None, max_flux=None):
+    """
+    Performs selections on source recarrays from
+
+    Args:
+        catalog_table: recarray of source catalog information.
+        lst_array: For coarse RA horizon cuts, lsts used in the simulation [radians]
+        time_array: Array of (float) julian dates corresponding with lst_array
+        latitude_deg: Latitude of telescope in degrees. Used for declination coarse horizon cut.
+        horizon_buffer: Angle (float, in radians) of buffer for coarse horizon cut. Default is about 10 minutes of sky rotation.
+                        Sources whose calculated altitude is less than -horizon_buffer are excluded by the catalog read.
+                        Caution! The altitude calculation does not account for precession/nutation of the Earth. A decent buffer is needed to
+                        ensure that the horizon cut doesn't exclude sources near the horizon.
```

Comment:
This worries me a little because how much buffer you need will get larger as a function of the time difference from J2000. Is there a way to be sure that we're always close enough?

Reply:
Is there some relation between time since J2000 and the accumulated effect of precession/nutation? I got the 10 minutes through trial and error in tests...

Reply:
Yes, it should. That makes me feel better, can you update the docstring to make it clear?